CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
+  * Added support for the service.beta.kubernetes.io/azure-load-balancer-tcp-idle-timeout annotation for Istio ingress gateways.
+  * Kubernetes Event-Driven Autoscaling (KEDA) is now supported in LTS.
+  * [Component Version API](https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster) is available for retrieving the exact versions of all AKS components in your cluster. This gives you real-time visibility into AKS components and add-ons, helping with proactive upgrade planning. Comparing component versions in your current version (cluster context) with potential K8s minor version updates to help identify possible breaking changes. CLI extension 18.0.0b19+ is required.
+
+* Preview Features
+  * [Azure Virtual Network Verifier](https://learn.microsoft.com/en-us/azure/virtual-network-manager/concept-virtual-network-verifier) is now available in Azure Portal (Nodepools blade) for troubleshooting outbound connectivity issues in your AKS cluster. 
+  * [Encryption in transit](https://learn.microsoft.com/en-us/azure/storage/files/encryption-in-transit-for-nfs-shares?tabs=azure-portal%2CUbuntu) is now available for the Azure File CSI driver for AKS version 1.33.
+  * Node Autoprovisioning metrics are now available through Managed Prometheus.
+  * Blue-green nodepool upgrade is now available in Preview in the v20250702preview API.
+ 
+* Bug Fixes
+  * Fixed [issue](https://github.com/Azure/AKS/issues/4720) where AKS evicted pods that had already been manually relocated, causing upgrade failures. This fix adds a node consistency check to ensure the pod is still on the original node before retrying eviction.
+  * Fixed an issue with image pull secrets settings when embargo CVE toggle is enabled.
+
+* Behavior Changes
+  * The delete-machines API will only delete machines from the system nodepool if the system addon PDBs are respected.
+  * Application routing component Pods are now annotated with kubernetes.azure.com/set-kube-service-host-fqdn to automatically have the [API server's domain name injected into the pod](https://learn.microsoft.com/azure/aks/outbound-rules-control-egress#required-outbound-network-rules-and-fqdns-for-aks-clusters) instead of the cluster IP, to enable communication to the API server. This is useful in cases where the cluster egress is via a layer 7 firewall.
+  * Add node anti-affinity for FIPS-enabled nodes for retina-agent when pod-level metrics are enabled.
+  * Container Insights agents now have a memory limit of 750Mi (down from 4Gi).
+  * Migration to AzureLinux3 is now supported for all Linux OS SKUs.
+  * Advanced Container Networking Services (ACNS) pods now run with priorityClassName: system-node-critical, preventing eviction under node resource pressure and improving cluster security posture.
+  
+
+* Component Updates
```

Comment:
Add links to every call out as much as possible - GH, AKS docs or upstream docs or release notes of the component.